src/api-v1.js

```diff
@@ -414,6 +414,45 @@ api.declare({
 });
 
 api.declare({
+  method:     'delete',
+  route:      '/killWorker',
+  // actions with context=worker will a query of the form ?provisionerId=...&workerType=...&workerGroup=...&workerId=...
+  // https://docs.taskcluster.net/reference/platform/taskcluster-queue/docs/actions#context
+  query: {
```

Comment:
Query strings are only for continuation token and limit, so these inputs shouldn't be provided here.

Please use route parameters instead.